src/server/services/games/new-order.service.ts

```diff
@@ -834,51 +866,63 @@ export async function getImagesQueue({
     ? [...poolCounters.Templar, ...poolCounters.Knight]
     : poolCounters[player.rankType];
 
-  const ratedImages = await getRatedImages({ userId: playerId, startAt: player.startAt });
+  const ratedImages = await getRatedImages({
+    userId: playerId,
+    startAt: player.startAt,
+    rankType: player.rankType,
+  });
 
-  for (const pool of rankPools) {
+  for (const pool of shuffle(rankPools)) {
```

Comment:
Not sure your goal here, the reason we have priority is sot hat we can care for the most dangerous stuff right away

Reply:
Seb mentioned that we want to sprinkle in low pri images when going through the queue